Dockerfile

```diff
@@ -27,7 +27,7 @@ WORKDIR /app
 
 COPY --from=prod-deps-image /build/node_modules/ ./node_modules
 COPY --from=build-image /build/dist/ ./dist
-COPY .env.vagrant ./
+COPY .env.test ./
```

Comment:
To be fair it should not be copied if this is just test file.

Reply:
Oh okay, cool. I'll remove.